CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Changelog for NeoFS Node
 ## [Unreleased]
 
 ### Added
+- Automated system tests run
```

Comment:
I'm really not sure we need to update changelog for this. It's a completely internal thing and I'd expect to see things related to NeoFS deliverables (cli/ir/storage/adm/lens) here.